main-1/main.go

```diff
@@ -0,0 +1,140 @@
+package main
```

Comment:
Confusing. This folder is called main-1 but the component is test1 :) Keep it consistent ?

Reply:
as this is another module we need a main named package in it. Let me see if i can modify it to something else.

Reply:
will take the name change in last iteration.